src/templates/Navigator/index.js

```diff
@@ -5,41 +5,11 @@ import classnames from 'classnames';
 import Layout from 'atoms/Layout';
 import Col from 'atoms/Layout/Col';
 import StepProgress from 'molecules/StepProgress';
+import { Partial, separatePartials, partialRenderer } from 'atoms/Partial';
```

Comment:
@patrickkim Consider putting `Partial` into a new directory of helpers or something, and maybe even packaging them up into an RCL library like `lodash`... Making this an Atom I think is confusing.